templates/addons/aurora/mysql-cf.yml

```diff
@@ -0,0 +1,190 @@
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
+  Name:
+    Type: String
+    Description: The name of the service, job, or workflow being deployed.
+  # Customize your Aurora Serverless cluster by setting the default value of the following parameters.
+  {{logicalIDSafe .Name}}InitialDBName:
```

Comment:
I added "initial" just because it is literally the initial database that gets created in the db server. Since the customers will be able to create additional database, this db is not necessarily *the* db.

The way to add additional dbS would be to access the cluster and execute SQL queries such as `CREATE DATABASE aseconddb`

Reply:
I think the "initial" prefix can be removed. I wanted to stress that this is not the only database they can have but this is probably not necessary.